### ISSUE TITLE
Incorrect command to publish beta

### ISSUE THREAD

**Issue by justjam2013** [homebridge/homebridge-plugin-template#75]
**Describe The Bug:**
The plugin template README says:
> ```# publish to @beta```
> ```npm publish --tag=beta```

However that "=" causes `npm` to publish with tag `latest`.

**Expected behavior:**
The correct command should be:
```
npm publish --tag beta
```

**Comment by NorthernMan54** [comment]
I'm thinking this would be better if it leveraged the auto publish feature similar to what we do with Homebridge-config-ui and others

**Comment by justjam2013** [comment]
I agree. One of the requirements for validation is creating releases, so publishing from github when a release is created, kills two birds with one stone.
I have separate github actions that publish to beta and to latest, that I would be happy to contribute.

**Comment by bwp91** [comment]
this is done
